karton/classifier/classifier.py

```diff
@@ -84,11 +132,19 @@ def wrapper(content, mime):
 
     def process(self, task: Task) -> None:
         sample = task.get_resource("sample")
-        sample_class = self._classify(task)
+
+        sample_classes = []
+
+        if self.yara_rules:
+            sample_classes += self._classify_yara(task)
+
+        filemagic_classification =  self._classify(task)
+        if filemagic_classification:
+            sample_classes.append(filemagic_classification)
```

Comment:
This is kinda awkward but I think it's better than replacing the returned type everywhere in `.classify()` :thinking:

Reply:
LGTM